firefox-ios/Client/TermsOfServiceManager.swift

```diff
@@ -32,4 +45,75 @@ struct TermsOfServiceManager: FeatureFlaggable {
         DefaultGleanWrapper().setUpload(isEnabled: value)
         Experiments.setTelemetrySetting(value)
     }
+
+    // MARK: – Constants
+
+    private enum VersionRange {
+        static let minimum = "138.0"
+        static let maximum = "138.1"
+    }
+
+    private enum Experiment {
+        static let onboardingPhase4ID = "new-onboarding-experience-experiment-phase-4-ios"
+        static let controlBranch = "control"
+    }
+
+    // MARK: – Public API
```

Comment:
Nit: Lets add a ticket somewhere too with the intention of future cleanup

https://mozilla-hub.atlassian.net/browse/FXIOS-12249 

Looks fine from my end. I will let you address other comments before merging.